content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
+skip_seealso: true
+aliases:
```

Comment:
> please delete the alias. You are not moving content - this is new content. Aliases are only used for new content.



> Hi! Welcome to the Istio community. I have left several comments which will help improve the quality of the content that appears within https://istio.io. Please remove translations. We have translations teams that will execute the translations. You have not translated the documentation, and if you had, it would slow down your PR as each translation team would request changes. Please submit as English only unless you wish to translate work. If you do wish to translate work, please do so in an independent PR.
> 
> Cheers
> -steve

Hi @sdake ,

I really appreciate you helping me review this document so carefully, I have benefited a lot and refactored this guide according to your comments, and I also removed translations. Please help to review changes in this PR.

Thanks
Feynman

Reply:
> please delete the alias. You are not moving content - this is new content. Aliases are only used for new content.

The above was confusing. I had meant to say "Alaises are only used for moving content".

Reply:
@sdake 

Sorry I am also confused about here, I originally keep the same format in aliases as follows:
```
aliases:
    - /docs/setup/kubernetes/prepare/platform-setup/kubesphere/
    - /docs/setup/kubernetes/platform-setup/kubesphere/
```
Then I delete this snippet according to comment. Could you pls help to give an example for alias? 

Thanks.

Reply:
Delete the aliases section entirely as you are not moving content.

Reply:
> Delete the aliases section entirely as you are not moving content.

@sdake 

The aliases section has been entirely removed in previous commit,  please see the [changes](https://github.com/istio/istio.io/pull/6424/files#diff-cae54a2a3d795eaf223951e4277c8169R2).